tests/e2e/pages/securityTipsPage.js

```diff
@@ -0,0 +1,29 @@
+exports.SecurityPage = class SecurityPage {
+  constructor(page) {
+    this.page = page;
+    this.loginButton = page.locator('#sign-in-btn');
+    this.vpnBanner = page.locator('.vpn-banner');
+    this.homePageNavButton = page.locator(
+      '.desktop-menu [data-event-label="Home"]'
+    );
+    this.SecTipsNavButton = page.locator(
+      '.desktop-menu [data-event-label="Security Tips"]'
+    );
+    this.SecurityPageUI = page.locator('.security-tips .no-vertical-padding');
```

Comment:
nit: Why do these selectors have uppercase first char?
(`this.SecTipsNavButton` vs `this.secTipsNavButton`)

Reply:
yep, already updated.